test/parallel/test-whatwg-url-origin.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+const common = require('../common');
+const path = require('path');
+const url = require('url');
+const URL = url.URL;
+const { test, assert_equals } = common.WPT;
+
+if (!common.hasIntl) {
+  // A handful of the tests fail when ICU is not included.
+  common.skip('missing Intl');
+  return;
+}
+
+const request = {
+  response: require(path.join(common.fixturesDir, 'url-tests.json'))
+};
+
+/* eslint-disable */
+/* WPT Refs:
+   https://github.com/w3c/web-platform-tests/blob/8791bed/url/url-origin.html
+*/
+function runURLOriginTests() {
+  // var setup = async_test("Loading data…")
```

Comment:
I think just commenting them out would make it easier to update and easier to compare with the upstream, maybe just me though